File: python-backend/app/api/v1/health.py
from datetime import datetime, timezone
from flask import Blueprint, jsonify

health_bp = Blueprint('health', __name__)


@health_bp.route('/health')
def health_check():
    """Health endpoint — critical for load balancers and monitoring."""
    return jsonify({
        'status': 'healthy',
        'timestamp': datetime.now(timezone.utc).isoformat(),
        'version': '1.0.0',
        'service': 'infrawatch-python-backend',
    })

File: backend/src/routes/health.js
'use strict';
const { Router } = require('express');
const router = Router();

router.get('/', (_req, res) => {
  res.json({
    status: 'healthy',
    timestamp: new Date().toISOString(),
    service: 'infrawatch-backend',
  });
});

module.exports = router;
